viewer/settings.py

```diff
@@ -134,3 +141,36 @@
     RAVEN_CONFIG = {
         'dsn': SENTRY_DSN,
     }
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'json': {
+            '()': 'dockerflow.logging.JsonLogFormatter',
+            'logger_name': 'experiments-viewer',
+        },
+        'verbose': {
+            'format': '%(levelname)s %(asctime)s %(name)s %(message)s',
+        },
+    },
+    'handlers': {
+        'console': {
+            'level': 'DEBUG',
+            'class': 'logging.StreamHandler',
+            'formatter': 'verbose',  # TODO: Use 'json' for Dockerflow.
```

Comment:
Yesterday I was checking out django-configurations. I do like the class based approach to django settings. I may do that in a next iteration.